munit-diff/shared/src/main/scala/munit/diff/DiffOptions.scala

```diff
@@ -0,0 +1,31 @@
+package munit.diff
+
+class DiffOptions private (
+    val contextSize: Int,
+    val showLines: Boolean,
+    val obtainedAsStripMargin: Boolean,
+) {
+  private def privateCopy(
+      contextSize: Int = this.contextSize,
+      showLines: Boolean = this.showLines,
+      obtainedAsStripMargin: Boolean = this.obtainedAsStripMargin,
+  ): DiffOptions = new DiffOptions(
+    contextSize = contextSize,
+    showLines = showLines,
+    obtainedAsStripMargin = obtainedAsStripMargin,
+  )
+
+  def withContextSize(value: Int): DiffOptions = privateCopy(contextSize = value)
+  def withShowLines(value: Boolean): DiffOptions = privateCopy(showLines = value)
+  def withObtainedAsStripMargin(value: Boolean): DiffOptions =
+    privateCopy(obtainedAsStripMargin = value)
+}
+
+object DiffOptions
+    extends DiffOptions(
+      contextSize = 1,
+      showLines = false,
+      obtainedAsStripMargin = false,
```

Comment:
```suggestion
      obtainedAsStripMargin = true,
```
to keep consistent with what was before, no?

Reply:
turns out the "before" was a bit inconsistent. there were several cases when it was true but more cases when it was false. at least the number of failing tests with true was a lot larger than with false.

Reply:
ach, ok. makes sense to have a smaller impact.